lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
+            </div>
+          </div>
+        </div>
+
+
+        <div class="card list-card team-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Team</p>
+              <h3 class="card-title">Team Solar</h3>
+              <p class="card-description">Team description that may need to be capped at 140 chars or so.</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper">
+            <div class="card-meta-details">
+              <div class="meta-detail team-members">
+                <span class="member-count">12 / 15 Members</span>
+                <span class="member-thumbs">
+                  <img alt="AlexeiK" href="" />
+                  <img alt="Assam-S" href="" />
+                  <img alt="SadieDearest" href="" />
+                </span>
+              </div>
+              <p class="meta-detail project-relation"><span class="icon fa-list"></span> Project 1</p>
+              <p class="meta-detail team-location"><span class="icon fa-globe"></span> North America</p>
+              <p class="meta-detail team-language"><span class="icon fa-chat"></span> English</p>
+              <p class="meta-detail team-activity">Last activity: 10 minutes ago</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href="">View<span class="sr"> Team Solar Team</span></a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card list-card team-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Team</p>
+              <h3 class="card-title">Team Solar</h3>
+              <p class="card-description">Team description that may need to be capped at 140 chars or so.</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper">
+            <div class="card-meta-details">
+              <div class="meta-detail team-members">
+                <span class="member-count">12 / 15 Members</span>
+                <span class="member-thumbs">
+                  <img alt="AlexeiK" href="" />
+                  <img alt="Assam-S" href="" />
+                  <img alt="SadieDearest" href="" />
+                </span>
+              </div>
+              <p class="meta-detail project-relation"><span class="icon fa-list"></span> Project 1</p>
+              <p class="meta-detail team-location"><span class="icon fa-globe"></span> North America</p>
+              <p class="meta-detail team-language"><span class="icon fa-chat"></span> English</p>
+              <p class="meta-detail team-activity">Last activity: 10 minutes ago</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href="">View<span class="sr"> Team Solar Team</span></a>
```

Comment:
Is the action supposed to be card-meta-wrapper for list-cards? It looks funny on my branch. (I realize the topic cards are actually square; just using it as an example)
![screen shot 2015-06-24 at 11 32 42 am](https://cloud.githubusercontent.com/assets/6981284/8334241/d580b606-1a64-11e5-8d64-7714380fc7ac.png)